lib/opal/builder.rb

```diff
@@ -241,7 +241,15 @@ def tree_requires(asset, asset_path)
           abs_base_path = Pathname(abs_base_path)
           entries_glob  = Pathname(abs_tree_path).join('**', "*{.js,}.{#{extensions.join ','}}")
 
-          Pathname.glob(entries_glob).map { |file| file.relative_path_from(abs_base_path).to_s }
+          Pathname.glob(entries_glob).map do |file|
+            if file.extname == '.rb'
```

Comment:
`.rb` is not special. This can also be `.opalerb` or `.js.rb` or `.js` (everything supported by BuilderProcessors).